packages/components/src/local/coa-status-board/index.tsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react'
+import { DataTable } from '../organisms/data-table'
+import { Badge } from '../atoms/badge'
+import { MessageCustom } from '@serge/custom-types/message'
+import { CollaborativeMessageStates } from '@serge/config'
+// import { ChannelData } from '@serge/custom-types'
+
+/* Import Types */
+import Props from './types/props'
+
+/* Import Stylesheet */
+import styles from './styles.module.scss'
+
+import ChannelCoaMessageDetail from '../molecules/channel-coa-message-detail'
+
+/* Render component */
+export const CoaStatusBoard: React.FC<Props> = ({ rfiMessages, roles, channels, isRFIManager, isUmpire, onChange, role }: Props) => {
+  // produce dictionary of channels
```

Comment:
I'm pretty sure we just have one channel here, so we don't need the dictionary.

We don't need to include the `channel` in the status board, since the board is just for this channel.